source/agora/consensus/EnrollmentManager.d

```diff
@@ -130,44 +135,45 @@ public class EnrollmentManager
     {
         this.db.close();
         this.validator_set.shutdown();
+        this.enroll_pool.shutdown();
     }
 
     /***************************************************************************
 
-        Add a enrollment data to the validators set
+        Add a enrollment data to the enrollment pool
 
         Params:
             block_height = the current block height in the ledger
             finder = the delegate to find UTXOs with
             enroll = the enrollment data to add
 
         Returns:
-            true if the enrollment data has been added to the validator set
+            true if the enrollment data has been added to the the enrollment pool
 
     ***************************************************************************/
 
     public bool add (ulong block_height, scope UTXOFinder finder,
         const ref Enrollment enroll) @safe nothrow
     {
-        return this.validator_set.add(block_height, finder, enroll);
+        return this.enroll_pool.add(block_height, finder, enroll);
     }
 
     /***************************************************************************
 
         Returns:
-            the number of enrollments being managed by this EnrollmentManager.
+            the number of enrollments being managed by the enrollment pool.
             Note: this includes both registered and un-registered enrollments.
```

Comment:
I think the Note: should be removed, because the pool only tracks un-registered enrollments.